.babelrc

```diff
@@ -1,6 +1,3 @@
 {
-  "presets":["es2015"],
-  "plugins":[
-    "transform-object-rest-spread"
-  ]
+  "presets":["latest", "stage-1"]
```

Comment:
I figured while I was in the code might as well take advantage of a new feature babel has released. It's purely a convenience for future dev.